csharp/ql/src/semmle/code/csharp/exprs/Call.qll

```diff
@@ -295,7 +295,7 @@ class MethodCall extends Call, QualifiableExpr, LateBindableExpr, @method_invoca
 
   override Method getQualifiedDeclaration() { result = getTarget() }
 
-  override string toString() { result = "call to method " + this.getTarget().getName() }
+  override string toString() { result = "call to method " + concat(this.getTarget().getName()) }
```

Comment:
Do you want to include a separator of some kind in this concatenation? Otherwise the names will all be smooshed together.

Reply:
There is at most one name; `concat` is (a)bused to the case where there are none, in which case `concat` yields the empty string.

Reply:
I see.

/me wishes for expression-level `or`, yet again.